.github/scripts/utils.zsh/setup_linux

```diff
@@ -40,7 +40,7 @@ if (( ! deps_version )) {
 log_group 'Setting up pre-built Chromium Embedded Framework...'
 
 pushd ${project_root}/.deps
-local _filename="cef_binary_${deps_version}_${target//-/_}.tar.xz"
+local _filename="cef_binary_${deps_version}_${target//-/_}${deps_revision:+"_${deps_revision}"}.tar.xz"
```

Comment:
Doesn't this need a "v" prefix for revision?

Reply:
Correct, fixed.